.github/workflows/beam_PostCommit_Java_Tpcds_Flink.yml

```diff
@@ -101,5 +101,5 @@ jobs:
         with:
           gradle-command: :sdks:java:testing:tpcds:run
           arguments: |
-            -Ptpcds.runner=:runners:flink:1.17 \
+            -Ptpcds.runner=:runners:flink:1.18 \
```

Comment:
Sorry for not putting comments at once, but just realize that these github action changes will apply to both the master branch and the ongoing release process, but the current pending release-2.56.0 branch do not have `:runners:flink:1.18` project which will causing issue

To avoid issue we can leave .github/ content as is

Reply:
We might merge this after the release of 2.56.0?

Reply:
I'm in no rush to get this merged, so I'd be okay with that.